include/unifex/let_value_with_stop_token.hpp

```diff
@@ -266,23 +263,15 @@ struct _stop_token_operation<SuccessorFactory, Receiver, AlwaysVoid>::type {
           std::declval<inplace_stop_token>(),
           (Receiver2&&)r)))
     : func_((SuccessorFactory2 &&) func)
-    , receiver_token_(get_stop_token(r))
+    , receiverToken_(get_stop_token(r))
     , innerOp_(connect_inner_op(
           func_,
-          stop_source_.get_token(),
+          stopSource_.get_token(),
           (Receiver2 &&) r)) {}
 
-  ~type() {
-    if (stop_callback_needs_destruction_) {
-      callback.destruct();
-    }
-  }
 
   void start() noexcept {
-    callback.construct_with([this]() {
-      return stop_callback{std::move(receiver_token_), stop_source_};
-    });
-    stop_callback_needs_destruction_ = true;
+    stopSource_.register_callbacks(receiverToken_);
```

Comment:
I don't see a matching `deregister_callbacks`; do we need one?  I suppose it would go in the receiver's completion methods?